src/Bridge/Doctrine/Orm/Filter/AbstractFilter.php

```diff
@@ -173,4 +175,28 @@ protected function extractProperties(Request $request) : array
 
         return $request->query->all();
     }
+
+    /**
+     * @param array property
```

Comment:
I don't think we need to throw an exception here, is the exception for the people who want to use this function ?